pkg/cmd/repo/create/create.go

```diff
@@ -526,118 +745,63 @@ func interactiveLicense(client *http.Client, hostname string) (string, error) {
-	gitInit, err := git.GitCommand("init", path)
-	if err != nil {
-		return err
-	}
-	isTTY := io.IsStdoutTTY()
-	if isTTY {
-		gitInit.Stdout = io.Out
-	}
-	gitInit.Stderr = io.ErrOut
-	err = run.PrepareCmd(gitInit).Run()
-	if err != nil {
-		return err
-	}
-
-	gitRemoteAdd, err := git.GitCommand("-C", path, "remote", "add", "origin", remoteURL)
-	if err != nil {
-		return err
-	}
-	gitRemoteAdd.Stdout = io.Out
-	gitRemoteAdd.Stderr = io.ErrOut
-	err = run.PrepareCmd(gitRemoteAdd).Run()
-	if err != nil {
-		return err
-	}
-
-	if checkoutBranch == "" {
-		return nil
-	}
-
-	gitFetch, err := git.GitCommand("-C", path, "fetch", "origin", fmt.Sprintf("+refs/heads/%[1]s:refs/remotes/origin/%[1]s", checkoutBranch))
-	if err != nil {
-		return err
-	}
-	gitFetch.Stdout = io.Out
-	gitFetch.Stderr = io.ErrOut
-	err = run.PrepareCmd(gitFetch).Run()
-	if err != nil {
-		return err
-	}
-
-	gitCheckout, err := git.GitCommand("-C", path, "checkout", checkoutBranch)
-	if err != nil {
-		return err
-	}
-	gitCheckout.Stdout = io.Out
-	gitCheckout.Stderr = io.ErrOut
-	return run.PrepareCmd(gitCheckout).Run()
-}
-
-func interactiveRepoCreate(isDescEmpty bool, isVisibilityPassed bool, repoName string) (string, string, string, error) {
-	qs := []*survey.Question{}
-
-	repoNameQuestion := &survey.Question{
-		Name: "repoName",
-		Prompt: &survey.Input{
-			Message: "Repository name",
-			Default: repoName,
-		},
-	}
-	qs = append(qs, repoNameQuestion)
-
-	if isDescEmpty {
-		repoDescriptionQuestion := &survey.Question{
-			Name: "repoDescription",
+// name, description, and visibility
+func interactiveRepoInfo(defaultName string) (string, string, string, error) {
+	qs := []*survey.Question{
+		{
+			Name: "repoName",
 			Prompt: &survey.Input{
-				Message: "Repository description",
+				Message: "Repository Name",
+				Default: defaultName,
 			},
-		}
-
-		qs = append(qs, repoDescriptionQuestion)
-	}
-
-	if !isVisibilityPassed {
-		repoVisibilityQuestion := &survey.Question{
+		},
+		{
+			Name:   "repoDescription",
+			Prompt: &survey.Input{Message: "Description"},
+		},
+		{
 			Name: "repoVisibility",
 			Prompt: &survey.Select{
 				Message: "Visibility",
 				Options: []string{"Public", "Private", "Internal"},
 			},
-		}
-		qs = append(qs, repoVisibilityQuestion)
-	}
+		}}
 
-	answers := struct {
+	answer := struct {
 		RepoName        string
 		RepoDescription string
 		RepoVisibility  string
 	}{}
 
-	err := prompt.SurveyAsk(qs, &answers)
-
+	err := prompt.SurveyAsk(qs, &answer)
 	if err != nil {
 		return "", "", "", err
 	}
 
-	return answers.RepoName, answers.RepoDescription, strings.ToUpper(answers.RepoVisibility), nil
+	return answer.RepoName, answer.RepoDescription, strings.ToUpper(answer.RepoVisibility), nil
 }
 
-func confirmSubmission(repoName string, repoOwner string, inLocalRepo bool) (bool, error) {
+func interactiveSource() (string, error) {
+	var sourcePath string
+	sourcePrompt := &survey.Input{
+		Message: "Path to local repository?",
```

Comment:
Only prompts that are "yes/no" should end in a question mark. Others could end in something like `:` or no punctuation at all. Example: `Enter the path to the local repository: _`